osi_object.proto

```diff
@@ -61,6 +61,22 @@ message StationaryObject
         //
         optional Color color = 4;
 
+        // The IDs of the lanes that the stationary object is assigned to.
+        //
+        // \note Might be multiple if the stationary object is streching over multiple lanes.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated Identifier assigned_lane_id = 5;
+
+        // Percentage value of the total stationary object residing within a corresponding lane.
+        //
+        // \note Might be multiple if the stationary object is streching over multiple lanes.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated double assigned_lane_percentage = 6;
```

Comment:
Can't this be calculated out of base in comparison to lane width?
Or did I miss the point here? => Then a better description would be good

Reply:
Output from CCB - 03.03.2021:
1. Move _repeated Identifier assigned_lane_id_  and _repeated double assigned_lane_percentage_ to another pull request and consider adding them to a new "Static Object Classification" (see Moving Object Classification)